src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Symbology/FeatureLayerExtrusion/FeatureLayerExtrusion.xaml.cs

```diff
@@ -109,15 +109,15 @@ private void ChangeExtrusionExpression()
             RendererSceneProperties myRendererSceneProperties = myRenderer.SceneProperties;
 
             // Toggle the feature layer's scene properties renderer extrusion expression and change the button text
-            if (Button_ToggleExtrusionData.Content.ToString() == "Population Density")
+            if (ToggleButton.Content.ToString() == "Population density")
             {
                 myRendererSceneProperties.ExtrusionExpression = "[POP07_SQMI] * 5000";
-                Button_ToggleExtrusionData.Content = "Total Population";
+                ToggleButton.Content = "Total population";
```

Comment:
The UI for this sample is a bit ambiguous. It's not clear whether the button label indicates current status ("Now showing: total population") or shows an option that you can switch to ("Show total population instead").

Reply:
To be addressed in a future cross-platform PR.